packages/web-twig/src/Resources/components/Modal/README.md

```diff
@@ -26,10 +26,11 @@ accessible name for the dialog.
 
 ### API
 
-| Name      | Type     | Default | Required | Description                        |
-| --------- | -------- | ------- | -------- | ---------------------------------- |
-| `id`      | `string` | —       | ✔        | Modal ID                           |
-| `titleId` | `string` | `null`  | ✕        | ID of the title inside ModalHeader |
+| Name                   | Type     | Default | Required | Description                                                  |
+| ---------------------- | -------- | ------- | -------- | ------------------------------------------------------------ |
+| `closeOnBackdropClick` | `bool`   | `true`  | ✕        | Allows/disallows modal closure when clicking on the backdrop |
```

Comment:
```suggestion
| `closeOnBackdropClick` | `bool`   | `true`  | ✕        | Whether the modal will close when backdrop is clicked |
```

Only suggestion, we often use `Whether` word for boolean props to describe the possibility to something switch on/off or allow/disallow.

Reply:
changed to `Whether …` sentence